src/shape/bounding-sphere.js

```diff
@@ -99,10 +100,28 @@ pc.extend(pc, function () {
                 point.copy(ray.direction).scale(t).add(ray.origin);
 
             return true;
+        },
+
+        /**
+         * @function
+         * @name pc.BoundingSphere#intersectsBoundingSphere
+         * @description Test if a Bounding Sphere is inside this Bounding Sphere.
+         * @param {pc.BoundingSphere} sphere Bounding Sphere to test.
+         * @returns {Boolean} true if the Bounding Sphere is inside this Bounding Sphere and false otherwise.
+         */
+        intersectsBoundingSphere: function (sphere) {
+            var diff = diffBetweenPoints;
+            diff.sub2(sphere.center, this.center);
+            var totalRadius = sphere.radius + this.radius;
+            if (diff.lengthSq() <= totalRadius * totalRadius) {
+                return true;
+            }
+
+            return false;
         }
     };
 
     return {
         BoundingSphere: BoundingSphere
     };
-}());
+}());
```

Comment:
Please, ensure your IDE adds empty line at the end of the file as stated in our contribution document: https://github.com/playcanvas/engine/blob/master/CONTRIB.md#remove-all-trailing-spaces-and-ending-line